pkgs/tools/system/btop/default.nix

```diff
@@ -37,6 +39,11 @@ stdenv.mkDerivation rec {
     ${removeReferencesTo}/bin/remove-references-to -t ${stdenv.cc.cc} $(readlink -f $out/bin/btop)
   '';
 
+  postPhases = lib.optionals rocmSupport [ "postPatchelf" ];
+  postPatchelf = lib.optionals rocmSupport ''
+    patchelf --add-rpath "${rocmPackages.rocm-smi}/lib" $out/bin/btop
```

Comment:
When not using lib.makeLibraryPath, I think the correct thing to do is use lib.getLib: "${lib.getLib drv}/lib"

Reply:
Fixed :+1: 
For what its worth, `lib.getLib` resulted in the same output path as `rocmPackages.rocm-smi` is itself already a lib, i.e. does not produce a dedicated `lib` output

Reply:
Ok. By using getLib the code doesn't silently break if/when the package grows multiple outputs.